docs/references/config/config.toml.md

```diff
@@ -1720,6 +1720,21 @@ But it may have implications in the way the application rewards validators.
 Setting `timeout_commit` to `0s` means that the node will start the next height
 as soon as it gathers all the mandatory +2/3 precommits for a block.
 
+### consensus.skip_timeout_commit
+
+Start the next height as soon as the node gathers all the mandatory +2/3 precommits for a block.
+
+```toml
+skip_timeout_commit = false
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `false` |
+|                     | `true`  |
+
+Setting `skip_timeout_commit` to `true` has the same effect as setting [`timeout_commit`](#consensustimeout_commit) to `"0s"`.
```

Comment:
Can we add a sentence (or reword the current one) saying something like "do no use, it's deprecated and may be removed in future releases"